detectsex.rb

```diff
@@ -0,0 +1,64 @@
+require 'telegram/bot'
+require 'net/http'
+require 'json'
+
+file = File.open('tokens.txt')
```

Comment:
It's really complicated way to get token :) try to use ENV variables. and for example dotenv gem